gitlab/cli.py

```diff
@@ -308,6 +308,11 @@ def _get_base_parser(add_help: bool = True) -> argparse.ArgumentParser:
         action="store_true",
         default=os.getenv("GITLAB_SKIP_LOGIN"),
     )
+    parser.add_argument(
+        "--unmask-credentials",
+        help="Don't mask credentials in debug mode",
+        action="store_true",
```

Comment:
I think if we invert this the same way as `--no-ssl-verify`, we can make the the enable_debug call more elegant later?

```suggestion
        action="store_false",
```

Reply:
👍